zetta_utils/alignment/get_z_blocks.py

```diff
@@ -2,33 +2,20 @@
 
 from zetta_utils import builder, log
 from zetta_utils.bcube import BoundingCube
+from ..parsing.ngl_state import get_bcubes_from_annotations
+from ..parsing.ngl_state import load as load_ngl_layer
```

Comment:
I think importing from `zetta_utils.parsing.ngl_state` would be slightly better here -- it will be slightly easier for a potential user to figure out where it's going or copy the import line to another file. 
Doing relative imports is definitely fine for within-module imports, but when we're importing from another module I think it's better to stick to absolute.